crates/bevy_tasks/src/task_pool.rs

```diff
@@ -93,8 +93,13 @@ impl TaskPoolBuilder {
     }
 }
 
-/// A thread pool for executing tasks. Tasks are futures that are being automatically driven by
-/// the pool on threads owned by the pool.
+/// A thread pool for executing tasks. While futures usually need to be polled
```

Comment:
By convention, we have a one sentence "this struct in a nutshell", folllowed by an empty line, followed by more detailed explanation.

In this case, can you split the first sentence from the rest?